higlass/server.py

```diff
@@ -302,83 +316,101 @@ class Server:
     # Keep track of the server processes that have been started.
     # So that when someone says 'start', the old ones are terminated
     processes = {}
-    diskcache_directory = "/tmp/hgflask/dc"
 
     def __init__(
         self,
         tilesets,
-        port=None,
         host="localhost",
-        tmp_dir="/tmp/hgflask",
-        no_fuse: bool = False,
+        port=None,
+        name=None,
+        fuse=True,
```

Comment:
I am never sure myself but is this common practice? I typically assume if no argument was passed its value is falsy. That's why I called it `no_fuse`. I agree the double negation is also not optimal.

Reply:
I don't think it's uncommon to have flags with true as default. I'm actually anticipating that this becomes False by default in the future, or perhaps changing to a third value like `"infer"` to decide whether to use fuse by scanning the tilesets provided.

Reply:
Cool thanks for clarifying.
> I'm actually anticipating that this becomes False by default in the future
Looking forward to that :)